tests/fixtures/s3.py

```diff
@@ -165,6 +165,11 @@ async def get_big_s3_object(
             object_id="big-downloadable",
         )
 
+        # Cache computed 'content' property since it's inaccessible after temporary
+        # file is disposed, but the content is required for tests.
+        if not hasattr(object_fixture, "_content"):
```

Comment:
Why are you using a leading underscore here? This would mean the _content variable is private and should never be accessed except from the object itself. Maybe rather use plain "content" or if there is some collision that must be avoided use content_.

Reply:
I used underscore to distinguish cached content from actual "content" property, but you are right, this is also confusing.

Reply:
Sorry, markdown mangled some underscores. I updated my response.